docs/package.json

```diff
@@ -1,7 +1,8 @@
 {
 	"scripts": {
 		"dev": "mintlify dev",
-		"generate:dev": "wget https://staging-backend.composio.dev/openapi.json -O composio_openapi.json && pnpm generate",
+		"openapi:validate": "mintlify openapi-check composio_openapi.json",
+		"generate:docs": "curl -o composio_openapi.json https://backend.composio.dev/openapi.json && pnpm generate",
```

Comment:
Consider adding validation for the OpenAPI spec before generating documentation. The new `openapi:validate` script is added but not integrated into the documentation generation workflow. Consider updating the `generate:docs` script to:

```json
"generate:docs": "curl -o composio_openapi.json https://backend.composio.dev/openapi.json && pnpm openapi:validate && pnpm generate"
```

This will help catch any OpenAPI spec issues before they affect the documentation.